assets/scss/layouts/_header.scss

```diff
@@ -209,7 +209,6 @@ button#doks-versions {
   padding-left: 0.3125rem;
   font-size: $font-size-base * 0.75;
   color: $gray-700;
-  content: "Ctrl + /";
```

Comment:
I take it that this shortcut no longer works?

Reply:
Right; Elastic's JS gloms on to the input field and nullifies the listener for this shortcut